docs/authentication/enterprise-connections/account-linking.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Account linking
+description: Learn how Clerk handles account linking during Enterprise SSO.
+---
+
+Account linking is the process of connecting multiple user accounts from different services or platforms, allowing users to access
+various services with a single set of credentials. It enables seamless sign-in using Enterprise SSO alongside other authentication methods like username/password. Clerk automatically attempts to link accounts that share the same email address, assuming a single owner for each email.
+
+## How it works
+
+When a user attempts to sign in or sign up, Clerk checks if the email address from the Identity Provider (IdP) matches an existing account and attempts to link them. Email addresses from IdPs are considered verified by default.
+
+The following sections describe different scenarios that can occur during this process and explain how Clerk handles each one.
+
+![Flow chart of the SAML SSO account linking process in various scenarios.](/docs/images/authentication/account-linking-flow-saml.webp)
+
+### Email is verified in Clerk
+
+When a user signs into your app using an IdP that returns a matching verified email address, Clerk automatically links the Enterprise account to the existing account and completes the sign-in process. This includes accounts protected by passwords, as the Enterprise sign-in flow automatically bypasses password verification.
+
+### Email is not verified and verification isn't required
+
+By default, Clerk requires email verification at sign-up. For instances that have disabled this behavior, there is a possibility that an account may be created using an unverified email address.
+
+To configure email verification at sign-up:
+
+1. In the Clerk Dashboard, navigate to the [**Email, Phone, Username**](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) page.
+1. Next to **Email address**, select the settings icon and uncheck the **Verify at sign-up** toggle.
+
+When a user signs into your app using an IdP, Clerk automatically links the Enterprise account to the existing account by **also verifying the existing email address** and signing the user in. This includes accounts protected by passwords, as the Enterprise sign-in flow automatically bypasses password verification.
+
+### Email is not verified
```

Comment:
Is this what happens when "Email is not verified and verification is required" ?
If verification is required, and an email is not verified, then Clerk creates a completely new account?

Reply:
This is a great question. This lead @mzhong9723 and I down a bit of a rabbit hole testing this.

To answer your question:
- WHEN a user has an existing unverified email address (the account was created with raw email/password with "verify at sign-up" turned off)
- AND the application owner configured EASIE **or** SAML for a domain matching the user's existing email address
- AND the user signs in, and gets redirected to the IDP
- THEN that account linking occurs. We **do not** create a new account, but instead update the existing account's email address to be verified.

We suspect this is a reversion, and it's not EASIE-specific. We're tracking it here: https://linear.app/clerk/issue/ORGS-361/handle-preventing-account-linking-to-unverified-email-addresses